apiserver/common/networkingcommon/types.go

```diff
@@ -142,6 +144,8 @@ func BackingSubnetToParamsSubnet(subnet BackingSubnet) params.Subnet {
 	// lp:1672888
 	var spaceTag string
 	if subnet.SpaceName() != "" {
```

Comment:
If we use `network.DefaultSpaceName` here, we can delete the TODO above, I think.

Reply:
agreed.